collections/_release_4_5/entry-systems-xr-openxr-render-models.md

```diff
@@ -0,0 +1,24 @@
+---
+type: entry
+section: systems
+subsection: XR
+rank: 2
+importance: 3
+anchor: openxr-render-models
+title: OpenXR render models
+text: |
+  Godot added support for the `XR_EXT_render_model` and `XR_EXT_interaction_render_model` extensions introduced in OpenXR.
```

Comment:
```suggestion
  We've added support for the `XR_EXT_render_model` and `XR_EXT_interaction_render_model` extensions introduced in OpenXR!
```
Other cards refer to "we" so I think this is appropriate and flows a bit better/more conversationally